core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -292,6 +292,15 @@ public void updateInode(Supplier<JournalContext> context, UpdateInodeEntry entry
     mState.applyAndJournal(context, entry);
   }
 
+  /**
+   * @param inodeId id of inode to be updated
+   * @param accessTime the new access time
+   * @return the applied inode entry
+   */
+  public UpdateInodeEntry updateInodeAccessTime(long inodeId, long accessTime) {
+    return mState.applyInodeAccessTime(inodeId, accessTime);
```

Comment:
Current implementation can result in inconsistency in access time if master is interrupted before it has a chance to flush access time changes. The only way to guarantee journal updates is to turn of async update for access time. 

Regarding API design, I made this call specific to access time so user of the class cannot use it to introduce inconsistent journal for updating other more important attributes. To avoid ppl forget to add journal entry, the only way I can think of is to make the access time updater part of the inode tree class so the journal update will be managed exclusively within the API call. However I am concerned about adding attribute specific logic to InodeTree to make it overly complicated. What do you think?

Reply:
If this is going to be the non-journaling update, then should call this something like  `updateInodeAccessTimeNoJournal`? This a special case since this method will not journal like all the other methods.

Reply:
renamed.